### DISCUSSION TITLE
perspective-export-menu styles not loading & disable export and/or copy menu

### DISCUSSION THREAD

**Discussion by GREEB** [perspective-dev/perspective#1952] [Q&A] [answered]
Hello everyone,
 I'm having some trouble debugging ```perspective-export-menu``` and ```perspective-copy-menu``` not having ```dropdown-menu``` styles.

The menus don't have any styles. I never had this problem before and can't figure out why this happens.

no ```dropdown-menu``` styles
![image](https://user-images.githubusercontent.com/1221769/186933856-8b3f300f-c461-4537-a69a-fb63b2c00a1b.png)

Interesting fix: 
After pressing the "Reset" button next to the export-menu the menus styles appear
has ```dropdown-menu``` styles
![image](https://user-images.githubusercontent.com/1221769/186932522-cc9a3804-1aa7-4bc3-a6b9-9e7367963694.png)

Also is there any way to disable export and copy features for all Perspective?

Thanks for all your work!

**Comment by texodus** [comment, marked as answer]
I believe #1950 is the fix for this regression (though I experienced it on 2 _different_ modal windows ...), which is in v1.6.5 release I just pushed, can you please check this version and let me know if this persists?

If this is not fixed for you in 1.6.5, what _browser_ and _browser version_ are you using to repro the issue?

**Comment by GREEB** [comment]
I've just upgraded to 1.6.5 and it works perfectly!
i was not able to recreate this bug on any example finos hosts so i thought i did something to break this.

I had it on every os on every browser in my example. (Nextjs, react website)

Thank you so much for the fix!
